packages/dashboards/addon/components/dashboard-actions/export.js

```diff
@@ -1,53 +1,51 @@
 /**
- * Copyright 2017, Yahoo Holdings Inc.
+ * Copyright 2020, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  *
  * Usage:
- *   {{#dashboard-actions/export
- *      dashboard=dashboard
- *   }}
+ *    <DashboardActions::Export
+ *      @dashboard={{@dashboard}}
+ *      @title={{@dashboard.title}}
+ *      @disabled={{not @dashboard.validations.isTruelyValid}}
+ *    >
  *      Inner template
- *   {{/dashboard-actions/export}}
+ *    </DashboardActions::Export>
  */
 
-import { computed, get } from '@ember/object';
-import { dasherize } from '@ember/string';
-import ExportAction from 'navi-reports/components/report-actions/export';
-import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import MultipleFormatExport from 'navi-reports/components/report-actions/multiple-format-export';
 
-export default ExportAction.extend({
+export default class DashboardMultipleFormatExport extends MultipleFormatExport {
   /**
-   * @property {Service} naviNotifications
+   * @property {String} filename - filename for the downloaded file
+   * @override
    */
-  naviNotifications: service(),
+  @computed('dashboard.title')
+  get filename() {
+    return `${this.dashboard.title}-dashboard`;
+  }
 
   /**
-   * @property {String} href - API link for the report
+   * @property {undefined} csvHref - CSV export is not available for dashboards
+   * @override
    */
-  href: computed('dashboard', 'disabled', function() {
-    // Void the href on a should disabled
-    if (get(this, 'disabled')) {
-      return 'javascript:void(0);';
-    }
-
-    return `/export?dashboard=${get(this, 'dashboard.id')}`;
-  }),
+  csvHref = undefined;
 
   /**
-   * @property {String} download - suggested filename
+   * @property {Promise} pdfHref - Promise resolving to pdf download link
+   * @override
    */
-  download: computed('dashboard', function() {
-    return dasherize(get(this, 'dashboard.title')) + '-dashboard';
-  }),
+  @computed('dashboard.id')
+  get pdfHref() {
+    return Promise.resolve(`/export?dashboard=${this.dashboard.id}`);
+  }
 
   /**
-   * @method click
+   * @property {Array} exportFormats - A list of export formats
+   * @override
    */
-  click() {
-    get(this, 'naviNotifications').add({
-      message: `The download should begin soon.`,
-      type: 'info',
-      timeout: 'medium'
-    });
```

Comment:
are we getting rid of the notification here?

Reply:
No, the parent class is now `MultipleFormatExport`, will use its `{{on "click" (queue (fn this.notify exportFormat.type) ...`